Super basic style

SallyMcGrath opened the pull request:
- very minor html changes
- there's no point doing loads as we will reactify the thing
- super basic style thrown on
- moved the TOC off the view  - header footer menu and some buttons
- none of this is really ok but it's a little easier to parse, maybe
- a tiny improvement, I offer

gregdyke commented on exercises.js:
```diff
@@ -39,9 +39,10 @@ function findExercises() {
 function setupToc(exercises) {
   let $toc = document.getElementById("toc");
-  let $ul = document.createElement("ul");
+  let $ul = document.createElement("ol");
```
rename variable

gregdyke commented on exercises.js:
```diff
@@ -55,7 +56,7 @@ function setupToc(exercises) {
   for (let i = 0; i < exercises.length; i++) {
     let exercise = exercises[i];
-    let $nav = document.createElement("div");
+    let $nav = document.createElement("nav");
```
there's a nav html element??? who knew!!? (not me anyway)

SallyMcGrath replied:
BTW almost none of this html will remain in the React version as this DOM just doesn't make sense as is. But let's do what we can for now!

gregdyke commented on index.html:
```diff
@@ -1481,16 +1401,16 @@ <h2>Some more "projects"</h2>
       <h3>CYF Blocks is a project of <a href="https://codeyourfuture.io">Code Your Future</a></h3>
       <p>Recap your HTML/CSS knowledge with this <a href="https://scrimba.com/learn/htmlcss">free crash course</a> on Scrimba.</p>
     </footer>
+     <!-- <dialog aria-live="assertive" class="toast" id="toast" aria-labelledby="toast-title" closed>
```
toast?

SallyMcGrath replied:
If I get 20 minutes it will manifest there.

SallyMcGrath replied:
toast

gregdyke commented on style.css:
```diff
@@ -0,0 +1,218 @@
+  background: var(--theme-color--pop);
+  color: var(--theme-color--paper);
+}
+:focus {
```
this seems to also affect blockly blocks themselves. Not to mention leaving the last button pressed with a weird outline

SallyMcGrath replied:
sorted

gregdyke reviewed (changes requested):
Great start while waiting for React!

- Could we have less horizontal padding - it's losing some valuable blocks screen estate (at least on my not very big screen)
- Could we have a bit of vertical padding/margin around the actions buttons? They particularly get crowded by the output area
- "Exercises menu" (or something else more descriptive) rather than just "menu"?
- I think the focus/button outline needs a bit of tinkering. Leaving a button selected after it's clicked is a bit weird and it affects the input areas in blockly

Some kind of "copied to clipboard" popup is really missing after pressing share (I think it becomes a must have before the first course, but not needed as part of this change)

gregdyke reviewed (changes requested):
Amazing! Thanks for all the tweaks.

There is a small bug left: when I open the menu and select an exercise, the menu doesn't close (but the exercise is selected)

SallyMcGrath commented:
> Amazing! Thanks for all the tweaks.
> 
> There is a small bug left: when I open the menu and select an exercise, the menu doesn't close (but the exercise is selected)

fixed in this tiny commit

I also moved the buttons left

gregdyke commented:
I don't see flash or code for flash, but maybe I'm confused - I've had a frustrating day so I'm not at my brightest :)

gregdyke reviewed (approved):
wheee!
